examples/using-all-channels/app.php

```diff
@@ -55,5 +57,8 @@
     json_decode(getenv(NotifierInput::JIRA_USERS_TO_IGNORE), true)
 ));
 
+$loader = new FilesystemLoader(__DIR__ . '/templates');
+$twig = new Environment($loader);
```

Comment:
```$twig = new Environment(new FilesystemLoader(__DIR__ . '/templates'));```

Reply:
I tried to decouple from the template file but it does not make so much sense in this particular sense, I need the file itself to check if we get the same text, else it is not possible.
:smile:

Reply:
My comment was more referred to avoid the `$loader` variable: 
```php
$twig = new Environment(new FilesystemLoader('templates'));
```